src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoItem: FC<Props> = ({ todo }) => {
+  const { id, title, completed } = todo;
+
+  return (
+    <div
+      key={id}
+      className={cn('todo', { completed })}
+    >
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
```

Comment:
Add 'checked' prop to the input element, and set its value to 'completed'.

Reply:
Done, but React writing error, bcs I don't use onChange